openfisca_uk/parameters/statistics.yaml

```
@@ -0,0 +1,253 @@
+aggregate:
```

Aggregate could refer to either spend or participation
```suggestion
spend:
```

---

OK - though there are tax revenues later down, the name isn't super important though.

On the admin inclusion, I don't think these include administrative costs. It's not 100% clear because the benefit and caseload tables don't specify admin expenditure, but I did find some data on admin costs from the [2016-17 DWP accounts](https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/624319/dwp-annual-report-and-accounts-2016-2017.pdf#page=122), and the 2015-16 outturn data (which has zero administrative costs, as they're specified elsewhere) for individual benefits in those tables matches with the benefits and caseload tables figures. I think this means the admin costs are not included in these figures.